dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataverse/DataverseSaver.java

```diff
@@ -105,13 +103,10 @@
     // -------------------- PRIVATE --------------------
 
     private void sendSuccessNotificationAsync(Dataverse dataverse, User user) {
```

Comment:
Where is async happening now?

Reply:
Inside sendNotification method, this method was created when mail's were not async yet.